### ISSUE TITLE
implement functionality to do a statistical comparison across groups divided according to genotype

### ISSUE THREAD

**Issue by schoffelen** [schoffelen/MOUS#69]
As discussed, we will start with CNTNAP2. Genotype information is present. I have created a matlab-structure that can be used to match subject ID with genotype.
I plan to write a rather generic function to be able to do statistics on a comparison between genotypic groups for single condition or contrast data (MEG), both for oscillatory and time-domain responses.

**Comment by schoffelen** [comment]
It seems that there is consensus that NOW it's OK to start looking at this again, although some first want to establish a relationship between MEG and fMRI, because any correlation between genetics and MEG is deemed stronger. Just typing this as a reminder that the basic machinery (code) to test relations between MEG and genetics is almost in place.

**Comment by JuliaUdden** [comment]
We should not run associations without having discussed which associations
exactly, in which order, etc. at the MOUS meetings or workshop. However,
what we can do is to do literature search to build up an argument for doing
a particular MEG-candidate gene association, independent of the FMRI-MEG
results. The four paragraph subheading 'Dysfunctional oscillatory
processes' in the Poeppel and Giraud review from 2012 seems like a good
starting point for associations with KIAA.

**Comment by schoffelen** [comment]
Could you then please bring the discussion to a point that we actually can make decisions about this? For the past 3 years it has been discussed on and off, but nobody seems to really enforce some executive decision. If we continue in this vein, i.e. hitting the brake each time and mentioning that 'something should be discussed' without taking further action up until the point that a decision has been taken, we can wait with analyzing until we all retire (or get fired because of lack of output).

**Comment by annhul** [comment]
I thought it was were clearly discussed at the December meeting: the visual  erf main effect sentence vs list is significant after mcc and will be used as a genetic phenotype (said Simon). Before testing, I will evaluate whether it also correlates with fMRI (As instructed by Peter). Depending on if it does or not, we will decide which snp for group the particpants by (says Annika). The fmri-meg comparsion will be done by the april workshop and we can discuss the matter then. I will also prepare a suggestion of snp (or snp's) for that meeting. 
Has there been a change to  plan communicated to JM (?) by Peter or Simon (or why do you say theta there is consensus that NOW it's ok ?)

**Comment by annhul** [comment]
And to clarfy further, what has been given green light (as I understood it) is to test the parametric stuff in fmri as a cntnap replication (= the so called Tinike-Julia paper) but that is now considered a separate study and the tests there will not affect MOUS, yes?

**Comment by JuliaUdden** [comment]
I think the first message by Annika is a good plan, thanks for the
clarification. I would like to make a clarification on how I understood the
topic in the last message. I think we are still working in a staged way for
the functional paper, first figuring out how to report the replication
study. The next step there is that I will send the final MOUS results to
Tineke and then I anticipate that she will take the lead there. In a later
stage, we will use the interaction between the (RC+/RC-) and (sentence vs
word lists) manipulations, over sentences, since that is were there is high
similarity between the auditory and visual sample, and run an association
with the same CNTNAP2 SNP.

**Comment by nihll** [comment]
Reasons for not testing for variation in the MEG sample have been "there's only a limited number of SNPs that we can test" and "we have a small sample size for genetics". Well, we knew that from the beginning, so that's not particularly helpful now.   So, I think what Julia said a few comments ago makes sense " a literature search to build up an argument for doing a particular MEG-candidate gene association, independent of the FMRI-MEG results." 

I actually have done this before (early last year), granted that it needs to be updated given that a year has past. The point being that we should definitely discuss this in the April MOUS workshop, and not have this drag out for another year until some cool genetics results comes from fMRI and/or fMRI/MEG analyses. 

@JuliaUdden, I know your list of candidate genes have been revised several times, and perhaps I can check whether your most current list overlaps with mine - can I find that list somewhere on the big-U?

**Comment by JuliaUdden** [comment]
Annika has fmri-meg results and is working on the genetics literature
search, with some more info from me. Any literature search will be
strongest if made on the genes, and even SNPs we've already typed. I will
send around this info in an email tomorrow. If necessary for coordination
around this issue, perhaps Nietzsche can join our skype meeting next Monday
(which will probably be in the morning - Annika and I can do 10.00, what
about JM?)

onsdag 25 mars 2015 skrev Nihll <notifications@github.com>:

**Comment by schoffelen** [comment]
I cannot do 10 I have to teach. I suggest to do 14.00. Annika indicated that she could make it at that time.

I don't understand this sentence: "Any literature search will be
strongest if made on the genes, and even SNPs we've already typed"

**Comment by annhul** [comment]
I disagree with Nietzsche, for the reasons not to test for genetic variation in the MEG sample. The main reason for having waited with the genetic analyiss, is that we have not found a stable, stat. sig. phenotype in MEG. Now that we have a potential phenotype (sent vs seq main erf), Simon already said in the fall that we should move fwd. I will present this and the details related to the planned test in the workshop. So not to worry, the ball is in my court and I have it under controll.  

I believe Julia is referring partly to the cntnap2 replication with Tinike, but also to other studies in the group, no? 

@JuliaUdden there was a question about Amaia and some MMN results related to genetics (in an email or another issue, I can't remember where). The MMN is a very specific ERP, and our sentence/word level data is not comparable to the it for a number of reasons which I won't go into here. The short answer is thus, that I don't think we need to invite Amaia to the workshop, but I will have a look at the reference and I can meet with her while in Nijmegen if needed.

**Comment by nihll** [comment]
Okay, well I do not recall the sent vs. seq main ERF having been given the green light for genetics last fall, and nothing about it has been updated in the meetings since then.... I look forward to seeing the results! 

@annhul / @JuliaUdden, can you guys share the list of potential SNPs with me? The one I have from Julia is quite old.

**Comment by annhul** [comment]
The list depends on which analysis/paper we are talking about. You will hear everything in the workshop together with everyone else. The genetics part, as far as I am aware, was not planned to be part of your PhD. But off course, if you find a stable and reliable oscillatory response, you can suggest and motivate genetic contrast for that. The important part for you is now to focus on your chapters.

**Comment by schoffelen** [comment]
Why wait until the workshop?

**Comment by annhul** [comment]
Cause I haven't finalized it yet :) And I want to make a proper argument and not drop unconnected snippets of data/arguments. What's with the pressure? Don't you have your own stuff to worry about?

**Comment by nihll** [comment]
We tend to always wait until the workshop, and it the past the following has occured: One of us has an idea, and the other's support is, but the other's weren't prepared to see these results so they can't support it well enough. This leads to the idea being shot down, despite being a good idea. Then we have to try and present the idea X number of times until it gets approved.  I'm more in favour of showing progress / ideas along the way - in between meetings.
